nnpdfcpp/config/plotting.yml

```diff
@@ -0,0 +1,18 @@
+#
+# Configuration file for plotting options
+#
+
+############################################################
+uset0         : false   # use t0 in chi2check, validphys
+pdfcteq       : CT14nlo
+pdfmstw       : MMHT2014nlo68cl
+plotformat    : eps   # root file format
+errorband     : true     # 1 for 1sigma, 0 for 68% c.l.
```

Comment:
```suggestion
errorband     : true    # 1 for 1sigma, 0 for 68% c.l.
```

Reply:
@scarrazza @scarlehoff why do we have this file at all?